<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChinaControlWidget</class>
 <widget class="QWidget" name="ChinaControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>991</width>
    <height>23</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>9</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="SerialPortComboBox" name="comPortComboBox"/>
   </item>
   <item>
    <widget class="QComboBox" name="canBaudComboBox"/>
   </item>
   <item>
    <widget class="QComboBox" name="modeComboBox"/>
   </item>
   <item>
    <widget class="QPushButton" name="openButton">
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SerialPortComboBox</class>
   <extends>QComboBox</extends>
   <header>CanAdapter/SerialPortComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
